src/Actions/MoveCursor.ts

```diff
@@ -124,13 +129,12 @@ export class ActionMoveCursor {
                 anchor = active;
             }
 
-            return new Selection(anchor, active);
-        });
-        return Promise.all(promisedSelections)
-            .then((selections) => {
-                activeTextEditor.selections = selections;
-                return Promise.resolve(true);
-            })
-            .then(() => ActionReveal.primaryCursor());
+            selections.push(new Selection(anchor, active));
+        }
+
+        activeTextEditor.selections = selections;
+        await ActionReveal.primaryCursor();
```

Comment:
I guess this line should be actually `return ActionReveal.primaryCursor();` and then the `return true;` should be removed, right?

![Screen Shot 2020-06-09 at 17 24 38](https://user-images.githubusercontent.com/1935696/84167204-362cf400-aa76-11ea-868e-f85cf82fc3ad.png)

Reply:
Sure, that sounds reasonable enough. Thanks for the explanation.